kubernetes/pods/spinnaker-helm/walkthrough.md

```diff
@@ -0,0 +1,127 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [Spinnaker Walk Through](#spinnaker-walk-through)
+  - [Important Installation Considerations](#important-installation-considerations)
+  - [Installation](#installation)
+  - [Post installation step](#post-installation-step)
+    - [Enable permission to the service account](#enable-permission-to-the-service-account)
+    - [Create namespace for your deployment](#create-namespace-for-your-deployment)
+  - [Deploying and Scaling a Containerized Application through Spinnaker](#deploying-and-scaling-a-containerized-application-through-spinnaker)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Spinnaker Walk Through
+
+## Important Installation Considerations
+
+Spinnaker will allow you to orchestrate the continuous deployment of applications into your cluster, for this, Spinnaker need to have access to the docker registries of this applications.
+
+You need to specify the credentials and the path to which docker repositories you want to enable before installing in the [values.yaml](values.yaml#L15-L19)
+
+As you can see, by default the only enabled libraries are the following:
+
+```yaml
+accounts:
+- name: dockerhub
+  address: https://index.docker.io
+  repositories:
+    - library/alpine
+    - library/ubuntu
+    - library/centos
+    - library/nginx
+```
+
+## Installation
+
+For installing spinnaker just follow [this](README.md) instructions.
+
+## Post installation step
+
+### Enable permission to the service account
+
+Once installed, Spinnaker will require for us to grant it access to the cluster, for this we will create a ClusterRoleBinding:
+
+```bash
+$ kubectl apply -f ClusterRoleBinding.yaml
+clusterrolebinding "spinnaker" created
+```
+
+Note that this will give Spinnaker admin access to the cluster, you might want to limit the permissions in a production deployment.
+
+### Create namespace for your deployment
+
+Spinnaker cannot create namespaces, because of this we will first create a namespace for our deployments:
+
+```bash
+$ kubectl create namespace prod
+namespace "prod" created
+```
+
+## Deploying and Scaling a Containerized Application through Spinnaker
+
+Assuming that you have cli access to the cluster, you can access Spinnaker deck with:
+
+```bash
+export DECK_POD=$(kubectl get pods --namespace spinnaker -l "component=deck,app=spinnaker-spinnaker-spin" -o jsonpath="{.items[0].metadata.name}")
+kubectl port-forward --namespace spinnaker $DECK_POD 9000 &
+open http://localhost:9000
+```
+
+Once opened you will see the following screen:
+
+![screen shot 2018-08-10 at 12 59 23](https://user-images.githubusercontent.com/1569751/43954557-3882efe0-9c9d-11e8-8bfe-f1d51efeefe4.png)
```

Comment:
The images should all be checked into this repo

Reply:
done.

Reply:
Did you test this?  

Also, can you link to the original doc? The problem is that these things changes and once it does it just breaks.  We would like to make it easy to update next time.